static/app/views/settings/projectPlugins/index.spec.jsx

```diff
@@ -48,31 +56,39 @@ describe('ProjectPluginsContainer', function () {
       url: `/projects/${org.slug}/${project.slug}/plugins/github/`,
       method: 'DELETE',
     });
-    wrapper = mountWithTheme(
-      <ProjectPlugins params={params} organization={organization} />
+    render(
+      <ProjectPluginsContainer
+        plugins={{plugins, loading: false}}
+        params={params}
+        organization={organization}
+      />
     );
   });
 
   it('calls `fetchPlugins` action creator after mount', function () {
     expect(fetchPlugins).toHaveBeenCalled();
   });
 
-  it('calls `enablePlugin` action creator when enabling plugin', function () {
-    const onChange = wrapper.find('ProjectPlugins').prop('onChange');
+  it('calls `enablePlugin` action creator when enabling plugin', async function () {
+    const pluginItem = (await screen.findByText('Amazon SQS')).parentElement.parentElement
+      .parentElement;
+    const button = getByRole(pluginItem, 'checkbox');
```

Comment:
I've been using [`within`](https://testing-library.com/docs/dom-testing-library/api-within/) for this type of thing instead of directly importing the query methods